app/controllers/planning_applications_controller.rb

```diff
@@ -150,8 +150,15 @@ def edit_public_comment
 
   def recommend
```

Comment:
Future refactor - move all these actions into a nice restful `recommendations` controller ☺️

Reply:
I'm glad it wasn't just me thinking that.